lib/influxer/metrics/relation/where_clause.rb

```diff
@@ -59,7 +59,7 @@ def build_eql(key, val, negate)
       when Regexp
         "#{key}#{negate ? ' !~ ' : ' =~ '}#{val.inspect}"
       when Array
-        return build_none if val.empty?
+        return build_none(negate) if val.empty?
```

Comment:
That would be a breaking change, unfortunately( But that's the right way to handle empty values.

Please, add a change log entry with "breaking" in **bold**